SmartDeviceLink-iOS.podspec

```diff
@@ -47,6 +47,7 @@ ss.public_header_files = [
 'SmartDeviceLink/SDLAddSubMenu.h',
 'SmartDeviceLink/SDLAlert.h',
 'SmartDeviceLink/SDLAlertManeuver.h',
+'SmartDeviceLink/SDLButtonPress.h',
```

Comment:
The podspecs need to list every public file, based on looking at the pbxproj changes, most of the new public files are not included.